### ISSUE TITLE
Update new version

### ISSUE THREAD

**Issue by lxl-java** [PavlikPolivka/GitLabProjects#193]
When I attempted to use it in IDEA2023.3, I was unable to.

**Comment by ascheman** [comment]
Yes, it seems to be incompatible with newer IntelliJ versions (e.g., 2024.3):
<img width="268" alt="image" src="https://github.com/user-attachments/assets/2bd688d0-e48c-4ac0-9237-d43977accc52">
